set.mm

```diff
@@ -85209,8 +85209,8 @@ the first case of his notation (simple exponentiation) and subscript it
       wa a1i ssexd ) BDEZABCFZGZCHZABUCIZCHZJABCDKUEUGLUBUEUFUCMNZSZCHUGUDUICAB
       UCOPUFUHCQRTUA $.
 
-    $( The set of surjections between two classes exists.  (Contributed by AV,
-       8-Aug-2024.) $)
+    $( The set of surjections between two classes exists (without any
+       precondition).  (Contributed by AV, 8-Aug-2024.) $)
```

Comment:
I think it was more an appreciative comment from @tirix than a suggestion to change your comment.

Reply:
> I think it was more an appreciative comment from @tirix than a suggestion to change your comment.

First, I unserstood it in the same way. Later, however, I thought it would be a good hint to be documented in the comment itself.

Reply:
Both views are valid!